src/Entity/CalculationModification.php

```diff
@@ -18,54 +25,104 @@
         new Post(),
         new Put(),
         new Delete(),
-    ]
+    ],
+    formats: ['json'],
+    collectDenormalizationErrors: true
 )]
+#[ApiFilter(filterClass: SearchFilter::class, properties: ['payment'])]
 #[ORM\Entity(repositoryClass: CalculationModificationRepository::class)]
 class CalculationModification
 {
     #[ORM\Id]
     #[ORM\Column(length: 255, unique: true)]
```

Comment:
### 🤔 Question: Should we have here `#[Assert\NotBlank]` propery?

I sow it in `member` entity and think that all other entities should have the same